google/cloud/servicecontrol_v1/types/distribution.py

```diff
@@ -30,8 +30,8 @@ class Distribution(proto.Message):
 
       - the arithmetic mean of the samples
       - the minimum and maximum of the samples
-      - the sum-squared-deviation of the samples, used to compute variance
-      - a histogram of the values of the sample points
+      - the sum-squared-deviation of the samples, used to compute
+    variance   - a histogram of the values of the sample points
```

Comment:
@busunkim96 , I recall manually changing this file in order to release the client. I couldn't locate the root cause of this problem in the source files. Should I open an issue in https://github.com/googleapis/gapic-generator-python?